src/components/Input/BaseInput/BaseInput.types.ts

```diff
@@ -62,6 +62,11 @@ type BasicBaseInputProps = Omit<React.ComponentPropsWithRef<'input'>, 'size' | '
      */
     prefixComponents?: React.ReactElement[];
 
+    /**
+     * Component
```

Comment:
Can you change the description to be more descriptive? And we also should rephrase the description of `prefixComponents`